tools/publisher/src/main.rs

```diff
@@ -108,6 +108,15 @@ fn clap_app() -> clap::App<'static, 'static> {
                         .required(true)
                         .takes_value(true)
                         .help("version number to yank"),
-                ),
+                )
+                .arg(
+                    clap::Arg::with_name("location")
+                        .required(true)
+                        .takes_value(true)
+                        .long("location")
+                        .help("Path to `aws-sdk-rust` repo. The repo should be checked out at the \
+                               version that is being yanked so that the correct list of crate names \
+                               is used. This will be validated.")
+                )
```

Comment:
this actually needs to be `aws-sdk-rust/sdk`—should probably either update the help text or the tool

Reply:
Great catch! I brought back the git repository resolution logic and had it detect the case where it was given `aws-sdk-rust` to resolve `sdk/`.